docs/templates/getting-started/sequential-model-guide.md

```diff
@@ -137,6 +137,24 @@ model.compile(optimizer='rmsprop',
 # for a mean squared error regression problem
 model.compile(optimizer='rmsprop',
               loss='mse')
+
+# for custom metrics
+import keras.backend as K
+
+def mean_pred(y_true, y_pred):
+    return K.mean(y_pred)
+
+def precision_recall(y_true, y_pred):
+    precision = ...
+    recall = ...
+    return {
+        'precision': precision,
+        'recall': recall,
+    }
```

Comment:
Precision and recall cannot be computed in a batched way, they are global metrics.